src/cascade/stats/estimation.py

```diff
@@ -31,6 +31,47 @@ def meas_bounds_to_stdev(df):
     return df.drop(["meas_lower", "meas_upper"], axis=1)
 
 
+def es_to_stdev(mean_value, es):
+    """
+    Takes an array of values (like mean), and
+    effective sample size (es) and transforms it
+    to standard error assuming that the sample
+    size is Poisson distributed.
+
+    Args:
+        mean_value:
+        es:
+
+    Returns:
+
+    """
+    mean = mean_value.copy()
+    sample_size = es.copy()
+    count = mean * sample_size
+
```

Comment:
Don't need to copy the input arguments if they are unmodified by the function.

Reply:
- [ ] Don't copy values